requirements/pytorch/test.txt

```diff
@@ -14,3 +14,4 @@ psutil<5.9.4 # for `DeviceStatsMonitor`
 pandas>1.0, <1.5.2  # needed in benchmarks
 fastapi<0.87.0
 uvicorn<0.19.1
+tensorboard>=2.9.1, <2.11.0
```

Comment:
```suggestion
```
IMO, if tensorboardx becomes the new default, we should run it in our CI.

Reply:
Then we need to drop testing `log_graph`

Reply:
We would need to parametrize all TB tests so that they run once with each dependency. I suggest leaving this for follow-up work